src/parser/parse_tree.h

```diff
@@ -1444,7 +1444,9 @@ typedef enum
   PT_CHANGE_INDEX_COMMENT,
   PT_CHANGE_INDEX_STATUS,
   PT_ADD_MEMBERS,		/* alter user type */
-  PT_DROP_MEMBERS
+  PT_DROP_MEMBERS,
+  PT_SERIAL_OPTION,		/* alter serial type */
+  PT_SERIAL_COMMENT
```

Comment:
PT_SERIAL_COMMENT seems redundant. the comment column could be changed in the other codes (SERIAL_OPTION, CHANGE_OWNER). The behavior of corresponding code should be independent.

Reply:
We've fixed it, please confirm.

Reply:
Now that I think about it, I didn't need to add PT_SERIAL_COMMENT again.

I concluded that it was correct to remove PT_SERIAL_COMMENT because when only changing the COMMENT, it doesn't have any effect inside the switch statement and is modified externally, so I decided to remove it.

We plan to add the decache flag to the grammar part so that the decache logic can be checked in advance.